doc/nrf/ug_bootloader.rst

```diff
@@ -129,7 +141,7 @@ This can be done in the following way:
 
 * Using cmake::
 
-     cmake -GNinja -DBOARD=nrf52840dk_nrf5840 -Dmcuboot_OVERLAY_CONFIG=overlay-minimal-size.conf -DCONFIG_SECURE_BOOT=y -DCONFIG_BOOTLOADER_MCUBOOT=y ../
+     cmake -GNinja -DBOARD=nrf52840dk_nrf5840 -Dmcuboot_OVERLAY_CONFIG=overlay-minimal-external-crypto.conf -DCONFIG_SECURE_BOOT=y -DCONFIG_BOOTLOADER_MCUBOOT=y ../
```

Comment:
In the paragraph above this, change the Kconfig overlay file reference to overlay-minimal-external-crypto.conf?

Reply:
Fixed